### PR TITLE
Adds code for computing relative errors at higher frequencies for single pub models

### PR DESCRIPTION
**Pull request by matthewclegg**

### CODE REVIEW

File: src/driver/experimental_trial.py [hunk 1]
```diff
@@ -295,6 +296,18 @@ def _compute_single_publisher_fractions_dataframe(
             column_name = f"relative_error_at_{int(r*100):03d}"
             results[column_name] = [relative_error]
 
+        for f in range(2, max_frequency):
```
Comment (jiayu-google) [inline src/driver/experimental_trial.py:299]: How about f = 1? It seems not to hurt by recording f=1.

**Review by jiayu-google — APPROVED** [review]
LGTM with a minor comment.

**Comment by amiragha** [comment]
This looks good to me, having the same Q as Jiayu

File: src/driver/experimental_trial.py (hunk 1 above)
Comment (matthewclegg) [reply src/driver/experimental_trial.py]: The case f=1 was already covered by the previous code, so there was no need to duplicate it.  Nonetheless, I have changed this to start from f=1.

**Review by jiayu-google — APPROVED** [review]
LGTM, thanks Matt!

**Review by amiragha — APPROVED** [review]

**Pull request merged**